search_assistant/streamlit/app.py

```diff
@@ -167,7 +168,8 @@ def main() -> None:
         st.markdown('Get your SerpApi key [here]( https://serpapi.com)')
 
         if not are_credentials_set(additional_env_vars):
-            api_key, additional_vars = env_input_fields(additional_env_vars)
+            api_key: str = ''
+            api_key, additional_vars = env_input_fields(additional_env_vars, mode=api_key)
```

Comment:
is this line ok? I think `mode` would an actual mode value? just double checking

Reply:
Yes good point @snova-rodrigom 

@snova-jorgep - env_input_fields(additional_env_vars: List[str] = [], mode: str = 'SambaNova Cloud').

Can you take a look? Will hold off until you've checked this.

Reply:
right,  fixed